### ISSUE TITLE
Published on WordPress.org ?

### ISSUE THREAD

**Issue by sviluppomania** [PI-Media/json-api#34]
??
@pimedia

**Comment by pimedia** [comment]
Yes, there are plans to publish it WordPress soon, some code additions are needed. 
    On Saturday, March 12, 2022, 08:15:27 PM GMT+5, sviluppomania ***@***.***> wrote:  
 
 


??
@pimedia

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by sviluppomania** [comment]
?? @pimedia

**Comment by sviluppomania** [comment]
@pimedia  ??

**Comment by sviluppomania** [comment]
@pimedia ??

**Comment by pimedia** [comment]
I will start working on package and releasing it on WordPress. I hope they approve the submission. sorry for the delay.

**Comment by sviluppomania** [comment]
Hi @pimedia ,

Are there any updates and/or improvements to the plugin?

**Comment by sviluppomania** [comment]
@pimedia
